frontend/src/components/LevelMissionInfoBanner/LevelMissionInfoBanner.tsx

```diff
@@ -11,14 +12,21 @@ function LevelMissionInfoBanner({
 	openOverlay: () => void;
 }) {
 	return (
-		<button className="level-mission-info-banner" onClick={openOverlay}>
-			<span className="level-title-area">{`Level ${currentLevel + 1}`}</span>
+		<span className="level-mission-info-banner">
```

Comment:
@dhinrichs-scottlogic Would we be better off using some sort of landmark for this banner, or is it self-explanatory?

I'm wondering if "section" or "complementary" are appropriate. I do realise `<section>` is a bit controversial...
https://developer.mozilla.org/en-US/blog/aria-accessibility-html-landmark-roles/

Reply:
any semantic element is better than a `<span>`. 
If I follow [this flowchart](https://html5doctor.com/downloads/h5d-sectioning-flowchart.pdf),  `<article>` or `<section>` seems reasonable

Reply:
landmarks should be [used sparingly](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Region_role#accessibility_concerns). If we give this a section, with a header, a text paragraph and a button, I think there's enough context for it without giving it a landmark role, and you already can easily jump to its header. The first rule of aria is: don't use aria if you can avoid it, that's probably true for role declarations, too. 
mdn's comedians are putting it aptly here: 
![image](https://github.com/ScottLogic/prompt-injection/assets/125262707/da80b96b-4b00-4d54-8c9c-bcc877ab932c)

Reply:
I vote for `<article>`

Reply:
Well, that's a possibility. Another is that they are list items within a list inside the chat panel, which itself has role=log:
https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA23
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/log_role

The first link has exactly our use case, plus a working demo. I note once again that Narrator is not announcing "polite" additions to the conversation log, but NVDA is.

_Do we want to add this improvement as a separate issue?_

Reply:
I don't think that's our use case, the short mission info doesn't keep a log, or updates, it just contains different text between the levels, which is expected since we are changing context. The same is true for the handbook, control panel, level buttons... 
I'd say keep it as simple as possible, use either `<section>` or `<article>`, it's at least a little bit more semantic than `<span>`, but don't add any additional roles or aria for now.